JSDemos/Demos/TreeView/VirtualMode/description.md

```diff
@@ -1 +1,7 @@
-For working with a&nbsp;large amount of&nbsp;data, the TreeView provides the Virtual Mode. In&nbsp;this mode, the component loads a&nbsp;set of&nbsp;child nodes once their parent node is&nbsp;expanded. In&nbsp;this demo, the TreeView is&nbsp;associated with a&nbsp;remote data source. Note that the component shows a&nbsp;loading indicator, because data loading takes a&nbsp;long time.
\ No newline at end of file
+With virtual mode, the TreeView loads a node’s children when the node is expanded for the first time. This enhances performance on large datasets.
+
+To enable this feature, set the [virtualModeEnabled](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#virtualModeEnabled) property to **true**. Note that this mode is only available when the TreeView's [dataStructure](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#dataStructure) is plain.
+
+When the data source is remote, the TreeView requests data even for those nodes that do not nest others. To prevent this, set the [hasItems](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/items/#hasItems) field to **false** for the corresponding data objects. This example also uses the [hasItemsExpr](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#hasItemsExpr) property to specify a custom name for this field.
+
+As an alternative to virtual mode, you can use the [createChildren](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#createChildren) function to gain full control over data load. To see this approach in action, refer to the [Load Data on Demand](/Demos/WidgetsGallery/Demo/TreeView/LoadDataOnDemand) demo.
```

Comment:
` to gain full control` means what?

Reply:
To specify a function that processes the requested data in a custom way.

Reply:
'in a custom way' does not explain anything nor should you consider it as the correct way to express something in a technical context

Reply:
"As an alternative to virtual mode, you can use a custom logic to process requested data. To do this, specify the createChildren function as shown in the Load Data on Demand demo." - what about this?

Reply:
Ok